.github/workflows/yarn.yaml

```diff
@@ -1,11 +1,13 @@
 name: Check with yarn
-on: [push, pull_request ]
+on: [push, pull_request]
 jobs:
   build:
     name: Install and test with yarn
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
+        with:
+          node-version: "18"
```

Comment:
To get the tests to pass, you need to use Node.js 16.
- See https://github.com/cypress-io/request/issues/37